src/gt4sd/frameworks/molgx/src/molgx/ChemGenerator/ChemGraph.py

```diff
@@ -0,0 +1,2216 @@
+# -*- coding:utf-8 -*-
+"""
+ChemGraph.py
+
+Package for IBM Molecule Generation Experience
+
+MIT License
+
+Copyright (c) 2022 International Business Machines Corporation
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+from rdkit import Chem
+from rdkit.Chem.rdchem import BondType
+from rdkit import RDLogger
+
+from operator import attrgetter
+from collections import Counter, defaultdict
+import copy
+import re
+import itertools
+
+import logging
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+rdlogger = RDLogger.logger()
+
+
+class ChemVertex(object):
+    """Base class for a vertex in a chemical graph.
+
+    Attributes:
+        index (int): unique index of a vertex in a graph starting from 0
```

Comment:
docstrings are not following the style used in the library, this needs to be adapted to properly build the docs.

Reply:
the same applies to all the files, for examples check the other toolkit classes and functions